snap/snapcraft.yaml

```diff
@@ -295,12 +285,12 @@ parts:
       usr/lib/qemu/*: usr/bin/
       usr/share/qemu/*: qemu/
     override-pull: |-
-      [ "$(uname -m)" != "x86_64" ] && [ "$(uname -m)" != "aarch64" ] && exit 0
+      [ "$(uname -m)" != "x86_64" ] && [ "$(uname -m)" != "aarch64" ] && [ "$(uname -m)" != "s390x" ] && exit 0
```

Comment:
Not exactly sure why we have this line since I believe this will always exactly match up `build-on`...